cmd/config/subcommand/config/init_flags.go

```diff
@@ -4,13 +4,11 @@ package config
 var (
 	DefaultConfig = &Config{
 		Insecure: false,
-		Storage:  false,
 	}
 )
 
 //Configs
 type Config struct {
 	Host     string `json:"host" pflag:",Endpoint of flyte admin"`
 	Insecure bool   `json:"insecure" pflag:",Enable insecure mode"`
-	Storage  bool   `json:"storage" pflag:",Enable storage provider config"`
```

Comment:
I don't think we can remove that right? Because then it'll fail to parse existing config files in the wild?